specs2/specs2_junit.bzl

```diff
@@ -0,0 +1,16 @@
+load("//specs2:specs2.bzl", "specs2_repositories", "specs2_dependencies", "specs2_version")
+load("//junit:junit.bzl", "junit_repositories")
+  
+def specs2_junit_repositories():
+  specs2_repositories()  
+  junit_repositories()
+  # Aditional dependencies for specs2 junit runner
+  native.maven_jar(
+      name = "io_bazel_rules_scala_org_specs2_specs2_junit_2_11",
+      artifact = "org.specs2:specs2-junit_2.11:" + specs2_version(),
```

Comment:
As I mentioned in the PR notes the reason I added this was because I (mistakenly) used one version in the specs2.bzl file and a different one in the specs2_junit.bzl file. Finding that bug took me a few hours since the runtime error messages are really unclear. Once I found it out I realized my problem was code duplication so I solved it with a method.